File: PushNotificationFirebase_Message/app/src/main/java/com/example/pushnotificationfirebase_message/MyApplication.java
package com.example.pushnotificationfirebase_message;

import android.app.Application;
import android.os.Build;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        createChannelNotification();
    }
//đăng ký channel id cho notification
//    tiếp theo xử lý logic làm sao có thể nhận push notification cho firebase
    private void createChannelNotification() {
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.0){

        }
    }
}

File: TestXoa/app/src/main/java/com/example/testxoa/MyApplication.kt
package com.example.testxoa

import android.app.Application
import android.app.NotificationChannel
import android.app.NotificationManager
import android.os.Build

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        createChannelNotification()
    }

    private fun createChannelNotification() {
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            val channel = NotificationChannel(
                CHANNEL_ID, "Channel Service Exmple",
                NotificationManager.IMPORTANCE_DEFAULT
            )

            val manager = getSystemService(
                NotificationManager::class.java
            )

            manager?.createNotificationChannel((channel))
        }
    }

    companion object{
        const val CHANNEL_ID : String = "channel_service_example"
    }
}